.github/settings.yml

```diff
@@ -0,0 +1,87 @@
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: scorecard
+
+  # A short description of the repository that will show up on GitHub
+  description: Security Scorecards - Security health metrics for Open Source
+
+  # A URL with more information about the repository
+  homepage: ''
+
+  # Updates the default branch for this repository.
+  default_branch: main
+
+  # Either `true` to enable automated security fixes, or `false` to disable
+  # automated security fixes.
+  enable_automated_security_fixes: true
+
+  # Either `true` to enable vulnerability alerts, or `false` to disable
+  # vulnerability alerts.
+  enable_vulnerability_alerts: true
+
+# See https://docs.github.com/en/rest/reference/teams#add-or-update-team-repository-permissions for available options
+teams: []
+    # The permission to grant the team. Can be one of:
+    # * `pull` - can pull, but not push to or administer this repository.
+    # * `push` - can pull and push, but not administer this repository.
+    # * `admin` - can pull, push and administer this repository.
+    # * `maintain` - Recommended for project managers who need to manage the repository without access to sensitive or destructive actions.
+
+# Collaborators: give specific users access to this repository.
+# See https://docs.github.com/en/rest/reference/collaborators for available options
+
+# THIS LIST SHOULD ALWAYS MATCH TOP-LEVEL CODEOWNERS
+# https://github.com/ossf/scorecard/blob/main/.github/CODEOWNERS
+
+# TODO(owners): For ease of management, this should eventually shift to a
+#               defined GitHub team instead of individual usernames
+collaborators:
+  - name: azeemshaikh38
+    permission: admin
+
+  - name: inferno-chromium
+    permission: admin
+
+  - name: justaugustus
+    permission: admin
```

Comment:
Write now @justaugustus has `write permissions`.

Reply:
Should be ok, @justaugustus has enough work on scorecards to justify this. you all should reeval later if we need that many admins or switch to 2 or something.